xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,487 @@
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec.depl.adm_conf.repos.scc">
+   <title>Update and Pool Repositories</title>
+   <para>
+    Update and Pool Repositories are required on the &lcm; server to set up and
+    maintain the &cloud; nodes. They are provided by &scc; and contain all
+    software packages needed to install &cloudos; and the extensions (pool
+    repositories). In addition, they contain all updates and patches (update repositories). Update
+    repositories are used when deploying the nodes that build
+    &cloud; to ensure they are initially equipped with the latest software
+    versions available.
+   </para>
+   <para>
+    The repositories can be made available on the &lcm; server using one or more of the
+    following methods:
+   </para>
+   <itemizedlist mark="bullet" spacing="normal">
+    <listitem>
+     <para>
+      <xref linkend="sec.depl.adm_conf.repos.scc.local_smt"/>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <xref linkend="sec.depl.adm_conf.repos.scc.alternatives"/>
+     </para>
+    </listitem>
+   </itemizedlist>
+   <sect2 xml:id="sec.depl.adm_conf.repos.scc.local_smt">
+    <title>
+     Repositories Hosted on an &smt; Server Installed on the &admserv;
+    </title>
+    <para>
+     When all update and pool repositories are managed by an &smt; server
+     installed on the &lcm; server (see <xref linkend="app.deploy.smt_lcm"/>),
+     the &lcm; automatically detects all available repositories. No further
+     action is required.
+    </para>
+   </sect2>
+
+   <sect2 xml:id="sec.depl.adm_conf.repos.scc.alternatives">
+    <title>Alternative Ways to Make the Repositories Available</title>
+    <para>
+     If you want to keep your &cloud; network as isolated from the company
+     network as possible, or your infrastructure does not allow accessing a
+     &susemgr; or an &smt; server, you can alternatively provide access to the
+     required repositories by one of the following methods:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Mount the repositories from a remote server.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Synchronize the repositories from a remote server (for example via
+       <command>rsync</command> and cron).
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+        Manually synchronize the update repositories from removable media.
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     The repositories must be made available at the
+     default locations on the &lcm; server as listed in <xref
+     linkend="tab.depl.adm_conf.local-repos"/>.
+    </para>
+   </sect2>
+  </sect1>
+
+ <sect1 xml:id="sec.deploy.repo_locations">
+ <title>Repository Locations</title>
+<para>
+The following tables show the locations of all repositories that can be used for SUSE OpenStack Cloud.
```

Comment:
I guess SUSE OpenStack Cloud needs to be replaced with some entity?

Reply:
Fixed